source/scenario/introScreenScenario.vwf.yaml

```diff
@@ -17,7 +17,7 @@ extends: source/scenario/scenario.vwf
 properties:
   scenarioName: introScreenScenario
 
-  nextScenarioPath: "scenario1a"
+  nextScenarioPath: "scenario1h"
```

Comment:
This is just a branch to demo the blockly functionality on Tuesday - definitely dont want to check this in haha :)